src/miral/custom_decorations.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "miral/custom_decorations.h"
+#include "decoration_manager_adapter.h"
+
+#include "miral/decoration_manager_builder.h"
+
+#include "mir/options/option.h"
+#include "mir/server.h"
+#define MIR_LOG_COMPONENT "CustomDecorations"
```

Comment:
I don't see any need to override the default (`MIR_LOG_COMPONENT_FALLBACK="miral"`)

Reply:
This is a remnant from when I had this file as an external example class. Thanks for pointing it out.